Packs/VulnDB/Integrations/VulnDB/VulnDB.py

```diff
@@ -360,22 +360,40 @@ def vulndb_get_cve_command(args: dict, client: Client):
     cve_id = args['cve_id']
     max_size = args.get('max_size')
 
-    res = client.http_request(f'/vulnerabilities/{cve_id}/find_by_cve_id', max_size)
-    results = res.get("results")
+    response = client.http_request(f'/vulnerabilities/{cve_id}/find_by_cve_id', max_size)
+    results = response.get("results")
     if not results:
         return_error('Could not find "results" in the returned JSON')
     result = results[0]
     cvss_metrics_details = result.get("cvss_metrics", [])
+
     data = {
         "ID": cve_id,
         "CVSS": cvss_metrics_details[0].get("score", "0") if cvss_metrics_details else "0",
-        "Published": result.get('vulndb_published_date', '').rstrip('Z'),
-        "Modified": result.get('vulndb_last_modified', '').rstrip('Z'),
-        "Description": result.get("description", '')
+        "Published": result.get('vulndb_published_date').rstrip('Z'),
+        "Modified": result.get('vulndb_last_modified').rstrip('Z'),
+        "Description": result.get("description")
```

Comment:
why removed the '' ? 
If you will run it on None, you will get:
`AttributeError: 'NoneType' object has no attribute 'rstrip'`

Reply:
Right, my bad.

```suggestion
        "Published": result.get('vulndb_published_date', '').rstrip('Z'),
        "Modified": result.get('vulndb_last_modified', '').rstrip('Z'),
```